### DISCUSSION TITLE
登录后个人账户“动态”显示“网络请求错误或关注列表为空”

### DISCUSSION THREAD

**Discussion by SC68SC** [Paladinfeng/MiaoProject#218] [General]
请问如何解决?

**Comment by Paladinfeng** [comment]
更新App到最新版本

**Comment by se7entao** [comment]
什么时候能返场啊 新设备也想要安装

**Comment by Paladinfeng** [comment]
> 什么时候能返场啊 新设备也想要安装

现在可以从 Apple TV App Store 搜索到啊，新设备直接登陆旧帐号就可以直接下载

**Comment by se7entao** [comment]
好的 已经弄好了 感谢🙏

ZFXUE ***@***.***>于2024年4月30日 周二21:33写道：
